### PR TITLE
Add data libraries tutorial

### PR DESCRIPTION
**Pull request by hexylena**
- some edits of images from dagobah
- data lib tuto

### CODE REVIEW

File: topics/admin/tutorials/data-library/slides.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+
+# Data Libraries
+
+Provide a way to conveniently share Galaxy datasets within a group of Galaxy users or with everybody that has access to a specific instance of Galaxy.
+
+* Can import data from filesystem *without duplicating* it.
+
+--
```
Comment (nsoranzo) [inline topics/admin/tutorials/data-library/slides.html]: I'm not a big fun of incremental slides (unless there is a good reason), but maybe that's just me.

Comment (hexylena) [reply topics/admin/tutorials/data-library/slides.html]: nah same. just copied/pasted. I need to seriously reduce amount of text per slide too.

**Comment by hexylena** [comment]
xref https://github.com/galaxyproject/admin-training/issues/97

File: topics/admin/tutorials/ansible-galaxy/tutorial.md [hunk 2]
```diff
@@ -233,50 +214,28 @@ We have codified all of the dependencies you will need into a YAML file that `an
...
+>      version: 0.9.5
 >    - src: galaxyproject.nginx
 >      version: 0.6.4
 >    - src: galaxyproject.postgresql
 >      version: 1.0.2
 >    - src: natefoo.postgresql_objects
 >      version: 1.1
 >    - src: geerlingguy.pip
->      version: 2.0.0
+>      version: 1.3.0
```
Comment (abretaud) [inline topics/admin/tutorials/ansible-galaxy/tutorial.md:225]: Is it intended to downgrade these?

File: topics/admin/tutorials/data-library/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+Additionally we have a `user` directory, below the user directory are more directories with the user's email as they directory key. Data can be placed in here, and it will become accessible to those users (we set `user_library_import_dir: /libraries/user` for this.)
+
+![An add datasets dropdown menu in galaxy showing the options from history, from user directory, and under admins only, from import directory](../../images/data/import-menu.png)
+
+## from History
+
+This is by far the easiest and most convenient option for small datasets, or datasets that are just already in a history
+
+![A select box is shown listing files in the user's history](../../images/data/import-admin.png)
```
Comment (abretaud) [inline topics/admin/tutorials/data-library/tutorial.md:133]: Maybe zoom to show only the box or reduce the browser window width? Some images is very large making it hard to read without clicking on the image

File: topics/admin/tutorials/data-library/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+>    ```bash
+>    library name: mouse sequencing project
+>    library already exists! id: f597429621d6eb2b
+>    ```
+>
+{: .hands_on}
+
+That's it! It's quite easy. Note that we've used some special flags here, `--training` and `--legacy`. Training sets some defaults that make sense for the GTN (mostly around library descriptions / etc.)
+
+![Screenshot of data libraries, we're in a library folder named "Mouse sequencing project" and a directory and single file are shown. The file has an ugly URL as its name](../../images/data/imported.png)
```
Comment (abretaud) [inline topics/admin/tutorials/data-library/tutorial.md:241]: Move the image before or after? It's in the middle of the --legacy explanation currently

**Review by abretaud — COMMENTED** [review]
Looks good, just a few comments!

File: topics/admin/tutorials/ansible-galaxy/tutorial.md (hunk 2 above)
Comment (hexylena) [reply topics/admin/tutorials/ansible-galaxy/tutorial.md:225]: no

File: topics/admin/tutorials/data-library/tutorial.md (hunk 3 above)
Comment (hexylena) [reply topics/admin/tutorials/data-library/tutorial.md:133]: Cropped :)

File: topics/admin/tutorials/data-library/tutorial.md (hunk 4 above)
Comment (hexylena) [reply topics/admin/tutorials/data-library/tutorial.md:241]: fixed, I think.

**Comment by hexylena** [comment]
@abretaud rebased without the ansible-galaxy changes, not sure how those got there. Whenever you're happy with the changes, please feel empowered to merge :)

**Review by abretaud — APPROVED** [review]

**Comment by hexylena** [comment]
Thanks @abretaud

**Pull request merged**